package.json

```diff
@@ -29,7 +29,29 @@
     "./logger": "./lib/logger/index.js",
     "./logger/compat": "./lib/logger/compat.js",
     "./lib/logger": "./lib/logger/index.js",
-    "./lib/logger/compat": "./lib/logger/compat.js"
+    "./lib/logger/compat": "./lib/logger/compat.js",
+    "./v2": "./lib/v2/index.js",
+    "./v2/options": "./lib/v2/options.js",
+    "./v2/https": "./lib/v2/providers/https.js"
+  },
+  "typesVersions": {
```

Comment:
This fixes the submodule exports for typescript. TS imports don't actually support submodule exports yet, but typesVersion allows us to do a rewrite of the lookup paths for .d.ts files. The Admin and Web SDK use this hack.